### ISSUE TITLE
Acceptance of Terms and Conditions with publication of data

### ISSUE THREAD

**Issue by acka47** [hbz/oerworldmap#1069]
Depends on #1068 .

- [ ] Rename "Save" button to "Publish"
- [ ] Add check box with something like "I accept the Terms & Conditions" next to publish button.

**Comment by acka47** [comment]
DUplicate of #748. Closing.
